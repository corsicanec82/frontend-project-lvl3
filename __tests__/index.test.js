import fs from 'fs';
import path from 'path';
import { html } from 'js-beautify';

const fixuturesPath = path.join(__dirname, '__fixtures__');
const getTree = () => html(document.body.innerHTML);

beforeAll(() => {
  const initHtml = fs.readFileSync(path.join(fixuturesPath, 'index.html')).toString();
  document.documentElement.innerHTML = initHtml;
});

test('application', () => {
  expect(getTree()).toMatchSnapshot();
});
