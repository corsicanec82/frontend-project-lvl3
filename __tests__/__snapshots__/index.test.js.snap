// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application 1`] = `
"<div class=\\"container\\">
    <div class=\\"jumbotron\\">
        <h1>Form to add RSS feed</h1><br>
        <form>
            <div class=\\"form-group row\\">
                <div class=\\"col\\"><label class=\\"sr-only\\" for=\\"inputRSS\\">label</label> <input type=\\"text\\" class=\\"form-control\\" id=\\"inputRSS\\" placeholder=\\"Enter data to add rss feed\\"></div>
                <div class=\\"col\\"><button type=\\"submit\\" class=\\"btn btn-primary\\">Add feed</button></div>
            </div>
        </form>
    </div>
</div>
<script src=\\"main.js\\"></script>"
`;
